src/components/app/data/utils.js

```diff
@@ -363,7 +363,6 @@ export const transformCourseEnrollment = (rawCourseEnrollment) => {
   const courseEnrollment = { ...rawCourseEnrollment };
   // Return the fields expected by the component(s)
   courseEnrollment.title = courseEnrollment.displayName;
-  courseEnrollment.microMastersTitle = courseEnrollment.micromastersTitle;
```

Comment:
[inform] Renamed the usage of `microMastersTitle` in components, etc. to `micromastersTitle`. The BFF will be returning `micromasters_title` (i.e. `micromastersTitle`). This transform is no longer needed now or with the BFF response.